Style.css

```diff
@@ -169,6 +183,106 @@ h1 {
   color: #172b4d;
 }
 
+/* pop-up menu */
+#showDetails {
+  display: none;
+}
+
+#showDetails.activate {
+  position: relative;
+  top: 8vh;
+  z-index: 1;
+  display: flex;
+  flex-direction: column;
+  width: 95%;
+  margin: 0 auto;
+  background-color: white;
+  border: 1px solid #ebecf0;
+  box-shadow: 0 48px 48px rgb(37 47 137 / 8%);
+  border-radius: 16px;
+}
```

Comment:
- [ ] According to the previous reviewer, Kindly make your image fit well to the work card as we have it in the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

Your design 👇 

![Screenshot 2022-12-22 092303](https://user-images.githubusercontent.com/78634841/209090234-302697ad-6c97-4280-83bd-ed88ed6c1152.png)

Figma design 👇 

<img src="https://user-images.githubusercontent.com/17243325/208959934-1e193ee9-09ff-4f6f-86d5-940a7ef55fd7.png">